src/main/java/frc/robot/subsystems/funnel/Ports.java

```diff
@@ -0,0 +1,12 @@
+package frc.robot.subsystems.funnel;
+
+public class Ports {
+    public static class Funnel{
+        /**
+         * input motor data.
+         * input boolean motor data.
+         */
+        public static final int MOTOR = 0;
+        public static final boolean IS_MOTOR_INVERTED = true;
```

Comment:
IMO documentation isn't needed here as the names of the constants are clear, but anyway, it's better to document every constant by itself, and not put the whole documentation together. 
Imagine you have dozens of constants in your list – if they will all be documented together, it will be very hard to read.